sidebar.shtml

```diff
@@ -12,6 +12,9 @@
       <li>
         <a href="/news">News: Latest information</a>
       </li>
+      <li>
+        <a href="/roadmap.html">Roadmap: What's planned</a>
+      </li>
```

Comment:
This gets the sidebar, which is fine.  How about a top navigation tab?  That necessitates adding a link in this list:

https://github.com/openssl/web/blob/68f71e63270307dde5343cfc5b0f31b49cbd3167/inc/banner.shtml#L23-L32

Reply:
Suggestion for inc/banner.shtml:

``` html
    <li><a href="/roadmap.html" title="What's planned">Roadmap</a></li>
```

Reply:
Okay, added.